index.html

```diff
@@ -4080,11 +4080,9 @@ <h3>Parsing content</h3>
 <!-- - - - - - - - - - - - - Security and Privacy - - - - - - - - - - - - - -->
 <section> <h2 id="security">Security and Privacy</h2>
   <p>
-    NFC technology involves multiple levels of security. Payments done with NFC
-    are considered to be secure at hardware level, but the whole software stack
-    needs to be security hardened.
-    However, NFC data transfers involve additional threats. In order to mitigate
-    these, the NFC Forum introduced [[NDEF-SIGNATURE]].
+    In order to protect the integrity and authenticity of NDEF messages, the NFC
+    Forum introduced [[NDEF-SIGNATURE]]. Signing NDEF records prevents malicious
+    use of NFC tags.
```

Comment:
but it is also the other way around.

This sounds like you harden the tag instead, but it also allows the reader to only accept such hardened tags and thus hardens the reader as well

Reply:
That's not how I intended to be understood ;)
How would you rephrase it?

Reply:
I would remove "Signing NDEF records prevents malicious use of NFC tags." 
That is inaccurate.

Reply:
I've removed it.